### ISSUE TITLE
1.1.x Helm Charts use release-1.1-latest-daily image

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#12713]
The release binary uploaded to Github uses the correct image tags, but the Helm repo does not.

```
wget https://storage.googleapis.com/istio-release/releases/1.1.0/charts/istio-1.1.0.tgz
tar xvf istio-1.1.0.tgz
$ grep tag istio/values.yaml
  # Default tag for Istio images.
  tag: release-1.1-latest-daily
```

This will impact anyone using Helm from the repo (`helm install istio.io/istio ...` vs `helm install install/kubernetes/helm/istio ...`)

@utka @duderino @mandarjog

**Comment by howardjohn** [comment]
Tested on 1.1rc1, 1.1.0, 1.1.1 draft. All have the same issue. 1.0.6 did not have this issue since we didn't have a Helm repo I think.

**Comment by kyessenov** [comment]
What is helm repo? Is this published somewhere outside this repository?

**Comment by howardjohn** [comment]
https://storage.googleapis.com/istio-release/releases/1.1.0/charts/

Under "Helm Chart Index" in releases

**Comment by kyessenov** [comment]
I've never used `helm install`. Is this an issue with our CI?

**Comment by kyessenov** [comment]
I'm seeing "the specified key does not exist" when I follow that link.

**Comment by howardjohn** [comment]
```
helm repo add istio.io https://storage.googleapis.com/istio-release/releases/1.1.0/charts/
helm install istio.io/istio --name istio --namespace istio-system
helm fetch istio/io/istio
```
https://storage.googleapis.com/istio-release/releases/1.1.0/charts/index.yaml describes what can be downloaded, helm will use this. So it shows the 3 charts we have, istio, istio-cni, istio-init.

https://storage.googleapis.com/istio-release/releases/1.1.0/charts/istio-1.1.0.tgz

**Comment by kyessenov** [comment]
Hmm, I've started looking for a place where this is published but so far no luck.

**Comment by howardjohn** [comment]
release/gcb/github_publish_release.sh:69 Indicates where we publish it on github - added in https://github.com/istio/istio/pull/9496/files

https://github.com/istio/istio/blob/master/release/gcb/helm_values.sh is what we need to run on https://github.com/istio/istio/blob/master/release/gcb/helm_charts.sh maybe?

**Comment by mandarjog** [comment]
This has never worked, all RCs also pointed to the latest daily...
I am fixing 1.1.0 and 1.1.1 charts now.

**Comment by kyessenov** [comment]
@mandarjog is this a manual fix? are we using GCB for helm repo?

**Comment by mandarjog** [comment]
Yes,  I fixed 1.1.0 release manually to point to the correct tag.

however 1.1.1 charts are not correct including its name, so I am not fixing them since 1.1.1 is not released yet.

**Comment by howardjohn** [comment]
I verified 1.1.0 looks good Mandar, thanks for the fix

**Comment by howardjohn** [comment]
Actually, think it still has issues. The tag should be `1.1.0` not `release-1.1.0`

**Comment by mandarjog** [comment]
fixed.

**Comment by howardjohn** [comment]
@mandarjog this is fixed in 1.1.0 but not 1.1.1

**Comment by howardjohn** [comment]
This has been fixed
